scout/build/variant/variant.py

```diff
@@ -249,6 +249,22 @@ def build_variant(
     variant_obj["local_obs_old_desc"] = variant.get("local_obs_old_desc")
     variant_obj["local_obs_old_nr_cases"] = variant.get("local_obs_old_nr_cases")
 
+    # local observations from cancer pipeline
+    variant_obj["local_obs_cancer_germline_old"] = variant.get("local_obs_cancer_germline_old")
```

Comment:
Here it is a bit confusing for me. Like what's the difference between `local_obs_cancer_germline_old` and `local_obs_cancer_germline_old_freq`etc? Could it be clarified either by comment or better by renaming these strings to something more user friendly? Is `local_obs_cancer_germline_old` the number of observarions? Then perhaps rename it to `local_obs_cancer_germline_old_obs`?

Reply:
Agree in principle, but I did want them consistent with the RD ones, and didn't want to change those keys. I'll add some more comments!